### DISCUSSION TITLE
🚀 Introducing Universal Actions for Adaptive Cards 🌐✨

### DISCUSSION THREAD

**Discussion by surbhigupta12** [MicrosoftDocs/msteams-docs#11396] [Announcements]
### ❓ What does this mean?

Microsoft Teams now features Universal Actions for Adaptive Cards, enhancing interactivity and consistency across different platforms and services. These Universal Actions enable users to interact with Adaptive Cards in a standardized way, regardless of where they are used. This functionality simplifies workflows, improves user experience, and enhances productivity by allowing consistent and seamless interactions within Teams and integrated applications.

The "EventPlanner" app is designed to simplify the planning, coordination, and management of events within Microsoft Teams. By using Universal Actions for Adaptive Cards, EventPlanner aims to provide a consistent and streamlined user experience for handling various interactions related to event management.

### 📑 Want to learn more? Here are some resources:

- The [docs](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/work-with-universal-actions-for-adaptive-cards?referrer=announcement) give you all the details.
- Here are [sample codes](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/work-with-universal-actions-for-adaptive-cards?referrer=announcement#code-samples) to get you started.

### ❔ Still have queries?

- Feel free to post your feedback here in the comments section.
- For bugs or deeper questions, raise an issue on the [documentation page](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/work-with-universal-actions-for-adaptive-cards#see-also) by selecting open documentation issue.

#### 📢 Subscribe for more updates: 
- [Configure to the RSS feed to get regular platform updates](https://learn.microsoft.com/en-us/microsoftteams/platform/whats-new?pivots=ga-feature#get-latest-updates)

### 💯 A massive thanks to everyone in the community. Your feedback and engagement have been valuable.

**Comment by soggy-cr0uton** [comment]
This has been addressed in PR #1320.

The issue was that values of 0 seconds (meaning "just announced" or "about to announce") were being displayed as "< 1m". The fix:
- Shows "now" for 0 seconds
- Adds hours formatting for longer durations (e.g., "1h 30m")
- Keeps "< 1m" for 1-59 seconds
